core/src/main/java/org/sonar/api/utils/log/Loggers.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * SonarLint Core - Implementation
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.api.utils.log;
+
+import javax.annotation.Nullable;
+import org.sonarsource.sonarlint.core.client.api.common.LogOutput;
+import org.sonarsource.sonarlint.core.log.LogOutputDelegator;
+import org.sonarsource.sonarlint.core.log.SonarLintLoggerFactory;
+import org.sonarsource.sonarlint.core.util.LoggedErrorHandler;
+
+public abstract class Loggers {
+  private static LogOutputDelegator logOutputDelegator = new LogOutputDelegator();
+  private static SonarLintLogger logger = new SonarLintLogger(logOutputDelegator);
+  private static SonarLintLoggerFactory factory = new SonarLintLoggerFactory(logger);
+
+  public static Logger get(Class<?> name) {
+    return logger;
+  }
+
+  public static Logger get(String name) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "name". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1172)